routers/posts.py

```
@@ -0,0 +1,79 @@
+    prefix="/posts",
+    tags=["posts"]
+)
+
+# 게시글 목록 조회 (페이지네이션 추가하기)
+@router.get("/", response_model=list[PostOut])
+def list_post():
+    return read_json("posts.json",default=[])
+
+# 게시글 검색
+@router.get("/",response_model=PostOut)
+def search_post(query=keyword):
```

Ruff (0.14.13)

Unused function argument: `query`

(ARG001)